tools/lit-stories/.storybook-shared/preview.css

```diff
@@ -0,0 +1,5 @@
+@chui lit-storybook;
```

Comment:
@dxos?

Reply:
Moving `@ch-ui/tokens` into the DXOS monorepo seems well outside the scope of this PR and I’d definitely like to discuss that first. I think these projects are meaningfully distinct and would benefit from being maintained separately.

Reply:
I needed to fix some of the packaging for `tokens` so I also changed its at-rule to be `@tokens`.